app/src/main/java/org/simple/clinic/monthlyReports/QuestionnaireType.kt

```diff
@@ -0,0 +1,47 @@
+package org.simple.clinic.monthlyReports
+
+import android.os.Parcelable
+import androidx.room.TypeConverter
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.util.room.SafeEnumTypeAdapter
+
+sealed class QuestionnaireType : Parcelable {
+
+  object TypeAdapter : SafeEnumTypeAdapter<QuestionnaireType>(
+      knownMappings = mapOf(
+          MonthlyScreeningReports to "monthly_screening_reports",
+          MonthlyDrugStock to "monthly_drug_stock"
```

Comment:
Is this type already being provided by the server? If not please remove this. We can add it once server has actual implementation of it.

Reply:
Data type of `questionnaire_type` changed to string and `QuestionnaireType.kt` deleted

Reply:
We would still need the questionnaire type class, I am just saying we don't need the drug stock type. Right now it's just screening reports.

Reply:
I have again added the questionnaire type and removed the monthly drug stock type.